express-post-json/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.listen(3000, () => {
+  // eslint-disable-next-line
+  console.log('Express server listening on port 3000');
+});
+
+const gradesArr = [];
+app.get('/api/grades', function (req, res) {
+  for (const key in grades) {
+    gradesArr.push(grades[key]);
+  }
+  res.json(gradesArr);
+});
```

Comment:
If you send 3 GET requests in a row, you should receive the same output. Test this.